### ISSUE TITLE
Return true range result for searches without predir

### ISSUE THREAD

**Issue by rbrtmrtn** [CityOfPhiladelphia/ais#110]
When you search for `1299 2ND ST`, you get a `parsed` match type because there are no known addresses on either North or South 2nd St. However, if you search for `1299 N 2ND ST` you get a true range result. Should we return both North and South true range and/or nearest known address matches for  `1299 2ND ST` as well?

**Comment by Alexander-M-Waldman** [comment]
Yes, we should return both N & S signatures - I'm into nearest if we do it across the board w/ interpolated
